az-core/src/main/java/azkaban/Constants.java

```diff
@@ -572,6 +572,8 @@ public static class FlowProperties {
     public static final String AZKABAN_FLOW_SUBMIT_USER = "azkaban.flow.submituser";
     public static final String AZKABAN_FLOW_EXEC_ID = "azkaban.flow.execid";
     public static final String AZKABAN_FLOW_PROJECT_VERSION = "azkaban.flow.projectversion";
+
+    public static final String AZKABAN_FLOW_PRODUCTION_MARKER = "azkaban.flow.production.marker";
```

Comment:
In my opinion this is not reflective of what we are doing here. While "production" has some internal meaning, that meaning will surely drift over time. Additionally, this is an OS repo, and production has no external meaning.

Reply:
Talked offline. I kind of agree all the points here.
Made additional comment on property name to address the correct usage in the future.
Sorry this has done in a scrappy way in a short time constraint, but I feel we're better to fill the gap when it makes sense to each jobtype who is responsible to manage production access.